parquet-encoding/src/main/java/org/apache/parquet/column/values/bitpacking/Packer.java

```diff
@@ -86,6 +105,7 @@ private static Object getStaticField(String className, String fieldName) {
   static IntPackerFactory leIntPackerFactory = getIntPackerFactory("LemireBitPackingLE");
   static BytePackerFactory beBytePackerFactory = getBytePackerFactory("ByteBitPackingBE");
   static BytePackerFactory leBytePackerFactory = getBytePackerFactory("ByteBitPackingLE");
+  static BytePackerFactory leBytePacker512VectorFactory = null;
```

Comment:
Please add a comment to explain leBytePacker512VectorFactory will be initialized lazily.

Reply:
I have added some comments